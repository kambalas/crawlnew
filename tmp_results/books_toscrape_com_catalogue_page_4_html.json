[
  {
    "name": "The Nameless City (The Nameless City #1)",
    "price": "38.16",
    "error": false
  },
  {
    "name": "The Murder That Never Was (Forensic Instincts #5)",
    "price": "54.11",
    "error": false
  },
  {
    "name": "The Most Perfect Thing: Inside (and Outside) a Bird's Egg",
    "price": "42.96",
    "error": false
  },
  {
    "name": "The Mindfulness and Acceptance Workbook for Anxiety: A Guide to Breaking Free from Anxiety, Phobias, and Worry Using Acceptance and Commitment Therapy",
    "price": "23.89",
    "error": false
  },
  {
    "name": "The Life-Changing Magic of Tidying Up: The Japanese Art of Decluttering and Organizing",
    "price": "16.77",
    "error": false
  },
  {
    "name": "The Inefficiency Assassin: Time Management Tactics for Working Smarter, Not Longer",
    "price": "20.59",
    "error": false
  },
  {
    "name": "The Gutsy Girl: Escapades for Your Life of Epic Adventure",
    "price": "37.13",
    "error": false
  },
  {
    "name": "The Electric Pencil: Drawings from Inside State Hospital No. 3",
    "price": "56.06",
    "error": false
  },
  {
    "name": "The Death of Humanity: and the Case for Life",
    "price": "58.11",
    "error": false
  },
  {
    "name": "The Bulletproof Diet: Lose up to a Pound a Day, Reclaim Energy and Focus, Upgrade Your Life",
    "price": "49.05",
    "error": false
  },
  {
    "name": "The Art Forger",
    "price": "40.76",
    "error": false
  },
  {
    "name": "The Age of Genius: The Seventeenth Century and the Birth of the Modern Mind",
    "price": "19.73",
    "error": false
  },
  {
    "name": "The Activist's Tao Te Ching: Ancient Advice for a Modern Revolution",
    "price": "32.24",
    "error": false
  },
  {
    "name": "Spark Joy: An Illustrated Master Class on the Art of Organizing and Tidying Up",
    "price": "41.83",
    "error": false
  },
  {
    "name": "Soul Reader",
    "price": "39.58",
    "error": false
  },
  {
    "name": "Security",
    "price": "39.25",
    "error": false
  },
  {
    "name": "Saga, Volume 6 (Saga (Collected Editions) #6)",
    "price": "25.02",
    "error": false
  },
  {
    "name": "Saga, Volume 5 (Saga (Collected Editions) #5)",
    "price": "51.04",
    "error": false
  },
  {
    "name": "Reskilling America: Learning to Labor in the Twenty-First Century",
    "price": "19.83",
    "error": false
  },
  {
    "name": "Rat Queens, Vol. 3: Demons (Rat Queens (Collected Editions) #11-15)",
    "price": "50.40",
    "error": false
  }
]