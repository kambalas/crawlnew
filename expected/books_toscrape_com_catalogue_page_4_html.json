[
  {
    "name": "The Nameless City (The ...",
    "price": "38.16",
    "error": false
  },
  {
    "name": "The Murder That Never ...",
    "price": "54.11",
    "error": false
  },
  {
    "name": "The Most Perfect Thing: ...",
    "price": "42.96",
    "error": false
  },
  {
    "name": "The Mindfulness and Acceptance ...",
    "price": "23.89",
    "error": false
  },
  {
    "name": "The Life-Changing Magic of ...",
    "price": "16.77",
    "error": false
  },
  {
    "name": "The Inefficiency Assassin: ...",
    "price": "20.59",
    "error": false
  },
  {
    "name": "The Gutsy Girl: Escapades ...",
    "price": "37.13",
    "error": false
  },
  {
    "name": "The Electric Pencil: Drawings ...",
    "price": "56.06",
    "error": false
  },
  {
    "name": "The Death of Humanity: ...",
    "price": "58.11",
    "error": false
  },
  {
    "name": "The Bulletproof Diet: ...",
    "price": "49.05",
    "error": false
  },
  {
    "name": "The Art Forger",
    "price": "40.76",
    "error": false
  },
  {
    "name": "The Age of Genius: ...",
    "price": "19.73",
    "error": false
  },
  {
    "name": "The Activist's Tao Te ...",
    "price": "32.24",
    "error": false
  },
  {
    "name": "Spark Joy: An Illustrated ...",
    "price": "41.83",
    "error": false
  },
  {
    "name": "Soul Reader",
    "price": "39.58",
    "error": false
  },
  {
    "name": "Security",
    "price": "39.25",
    "error": false
  },
  {
    "name": "Saga, Volume 6 (Saga ...",
    "price": "25.02",
    "error": false
  },
  {
    "name": "Saga, Volume 5 (Saga ...",
    "price": "51.04",
    "error": false
  },
  {
    "name": "Reskilling America: Learning ...",
    "price": "19.83",
    "error": false
  },
  {
    "name": "Rat Queens, Vol. 3: ...",
    "price": "50.40",
    "error": false
  }
]